internal/providers/noop/noop.go

```diff
@@ -19,12 +19,20 @@ package noop
 import (
 	"github.com/coreos/ignition/v2/config/shared/errors"
 	"github.com/coreos/ignition/v2/config/v3_5_experimental/types"
+	"github.com/coreos/ignition/v2/internal/platform"
 	"github.com/coreos/ignition/v2/internal/resource"
 
 	"github.com/coreos/vcontext/report"
 )
 
-func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
+func init() {
+	platform.Register(platform.Provider{
+		Name:  "metal",
```

Comment:
Is this still `metal` at this point or is it `noop`? I saw it was renamed in a later commit

Reply:
The registration was always under the name `metal`, even though the Go package was originally `noop`.

Reply:
Oh ok, I had assumed it was along those lines but wanted to make sure.